packages/x-components/src/x-modules/query-suggestions/components/__tests__/query-suggestion.spec.ts

```diff
@@ -58,7 +58,8 @@ describe('testing query-suggestion component', () => {
       eventPayload: suggestion,
       metadata: {
         moduleName: 'querySuggestions',
-        target: button
+        target: button,
+        feature: 'query_suggestion'
```

Comment:
As for these suggestion like components the feature is hardcoded in the template, and TypeScript can't help us there, I would suggest to make him help us here, so in case we rename one feature, or remove it we can make the changes easily. **For example, by wrapping every `metadata` object in a typed `expect.objectContaining` call (in every component).**

```ts
   expect(listener).toHaveBeenCalledWith({
      eventPayload: suggestion,
      metadata: expect.objectContaining<Partial<WireMetadata>>({
        moduleName: 'querySuggestions',
        target: button,
        feature: 'query_suggestion'
      });
    });
```